Packs/CommonPlaybooks/Playbooks/playbook-Endpoint_Enrichment_-_Generic_v2.1_6_8.yml

```diff
@@ -0,0 +1,1268 @@
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+    continueonerrortype: ""
+  "34":
+    id: "34"
+    taskid: 968db0d2-282a-4a9b-8071-39de4a0b5b09
+    type: regular
+    task:
+      id: 968db0d2-282a-4a9b-8071-39de4a0b5b09
+      version: -1
+      name: Get- host information from McAfee ePO v2
+      description: Finds systems in the McAfee ePO system tree.
+      script: McAfee ePO v2|||epo-find-system
+      type: regular
+      iscommand: true
+      brand: McAfee ePO v2
+    nexttasks:
+      '#none#':
+      - "4"
+    scriptarguments:
+      searchText:
+        complex:
+          root: inputs.Hostname
+      verbose:
+        simple: "false"
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 580,
+          "y": 890
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: true
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+    continueonerrortype: ""
+  "35":
+    id: "35"
+    taskid: f2dbaff5-7c92-47ad-80cc-991bfd80ff98
+    type: title
+    task:
+      id: f2dbaff5-7c92-47ad-80cc-991bfd80ff98
+      version: -1
+      name: Endpoint Reputation
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "36"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -790,
+          "y": 580
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "36":
+    id: "36"
+    taskid: 50fed99c-1eb9-4a6f-85d0-f9d5ee74bc5a
+    type: condition
+    task:
+      id: 50fed99c-1eb9-4a6f-85d0-f9d5ee74bc5a
+      version: -1
+      name: Should use !endpoint command?
+      description: Check if should run endpoint reputation command
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "4"
+      "yes":
+      - "37"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              complex:
+                root: inputs.UseReputationCommand
+            iscontext: true
+          right:
+            value:
+              simple: "True"
+          ignorecase: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -790,
+          "y": 720
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "37":
+    id: "37"
+    taskid: 79880978-7c28-4aaf-83b3-7f8ea42da67f
+    type: regular
+    task:
+      id: 79880978-7c28-4aaf-83b3-7f8ea42da67f
+      version: -1
+      name: Check Reputation
```

Comment:
![image](https://user-images.githubusercontent.com/28757135/227765129-86b8f775-7fd0-45ad-b949-5d72509f7d00.png)
Why reputation? it's an enrichment command

Reply:
I did not change this task. Erez worked on this task a few weeks ago.